INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:fk logss:- (PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3MAAzYE', file_size=1479, file_unique_id='AQADAckxG4yisVR4', height=51, width=90), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA20AAzYE', file_size=25478, file_unique_id='AQADAckxG4yisVRy', height=180, width=320), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3gAAzYE', file_size=141252, file_unique_id='AQADAckxG4yisVR9', height=450, width=800), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE', file_size=222042, file_unique_id='AQADAckxG4yisVR-', height=720, width=1280))
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:fk logss:- (PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3MAAzYE', file_size=1479, file_unique_id='AQADAckxG4yisVR4', height=51, width=90), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA20AAzYE', file_size=25478, file_unique_id='AQADAckxG4yisVRy', height=180, width=320), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3gAAzYE', file_size=141252, file_unique_id='AQADAckxG4yisVR9', height=450, width=800), PhotoSize(file_id='AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE', file_size=222042, file_unique_id='AQADAckxG4yisVR-', height=720, width=1280))
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6576956206
INFO:env_config:Created new user record for 6576956206
INFO:env_config:Callback 'svc_image' by 6576956206
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Text from 6576956206: Genrate a image of air conditi...
INFO:env_config:Upserting user 6576956206
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Request 3ebcca56-0e06-4adc-8e6a-e57d640aab6d created: image by 6576956206
INFO:env_config:Calling API: …
ERROR:env_config:API error: Request URL is missing an 'http://' or 'https://' protocol.
INFO:env_config:Request 3ebcca56-0e06-4adc-8e6a-e57d640aab6d updated: failed
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:/start by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_enhance_image' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_back' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_image' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_back' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_clean_audio' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:fk logss:- Audio(duration=10, file_id='CQACAgUAAxkBAAIBU2gV9SQqBNVl2yhLNsqOqNAtiVUFAALUGAACzTuxVJ_J-iBn6MikNgQ', file_name='Anchor_Nikola_Tesla.mp3', file_size=81261, file_unique_id='AgAD1BgAAs07sVQ', mime_type='audio/mpeg')
INFO:env_config:File MIME Type: audio/mpeg
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_clean_audio' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getFile "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: GET https://api.telegram.org/file/bot7580016374%3AAAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/photos/file_5.jpg "HTTP/1.1 200 OK"
ERROR:telegram.ext.Application:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_application.py", line 1298, in process_update
    await coroutine
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_handlers/basehandler.py", line 158, in handle_update
    return await self.callback(update, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lucky/py-ai/app.py", line 34, in enhance_image_callback
    api = EXTERNAL_APIS['image_enhancer']
          ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'image_enhancer'
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getFile "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: GET https://api.telegram.org/file/bot7580016374%3AAAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/photos/file_5.jpg "HTTP/1.1 200 OK"
ERROR:telegram.ext.Application:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_application.py", line 1298, in process_update
    await coroutine
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_handlers/basehandler.py", line 158, in handle_update
    return await self.callback(update, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lucky/py-ai/app.py", line 34, in enhance_image_callback
    api = EXTERNAL_APIS['image_enhancer']
          ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'image_enhancer'
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getFile "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: GET https://api.telegram.org/file/bot7580016374%3AAAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/photos/file_5.jpg "HTTP/1.1 200 OK"
INFO:env_config:Calling API: None
ERROR:env_config:API error: Invalid type for url.  Expected str or httpx.URL, got <class 'NoneType'>: None
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBBWgV3feFBPRIOYmTaidhRyGHTLxNAAL7yDEbjKKxVGbMHkNvy_OnAQADAgADeQADNgQ
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBBWgV3feFBPRIOYmTaidhRyGHTLxNAAL7yDEbjKKxVGbMHkNvy_OnAQADAgADeQADNgQ
ERROR:telegram.ext.Application:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_application.py", line 1298, in process_update
    await coroutine
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_handlers/basehandler.py", line 158, in handle_update
    return await self.callback(update, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lucky/py-ai/commands/handle_media.py", line 34, in handle_media
    [InlineKeyboardButton('🖼️ Enhance Image', callback_data='svc_enhance_image')]
TypeError: list indices must be integers or slices, not InlineKeyboardButton
INFO:env_config:/start by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140 (unknown username)
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
ERROR:telegram.ext.Application:No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_application.py", line 1298, in process_update
    await coroutine
  File "/opt/anaconda3/lib/python3.12/site-packages/telegram/ext/_handlers/basehandler.py", line 158, in handle_update
    return await self.callback(update, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/lucky/py-ai/commands/handle_media.py", line 34, in handle_media
    [InlineKeyboardButton('🖼️ Enhance Image', callback_data='svc_enhance_image')]
TypeError: list indices must be integers or slices, not InlineKeyboardButton
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:/start by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:env_config:Upserting user 6133968140
INFO:env_config:Callback 'svc_back' by 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBBWgV3feFBPRIOYmTaidhRyGHTLxNAAL7yDEbjKKxVGbMHkNvy_OnAQADAgADeQADNgQ
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getFile "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: GET https://api.telegram.org/file/bot7580016374%3AAAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/photos/file_6.jpg "HTTP/1.1 200 OK"
INFO:env_config:Calling API: None
ERROR:env_config:API error: Invalid type for url.  Expected str or httpx.URL, got <class 'NoneType'>: None
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBBWgV3feFBPRIOYmTaidhRyGHTLxNAAL7yDEbjKKxVGbMHkNvy_OnAQADAgADeQADNgQ
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Document from 6133968140
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Photo received: AgACAgUAAxkBAAIBCmgV3zFuSEH2y_U-SiuRBQra1O8EAAIByTEbjKKxVP__H4EAAcIn0gEAAwIAA3kAAzYE
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application is stopping. This might take a moment.
INFO:telegram.ext.Application:Application.stop() complete
INFO:__main__:Starting webhook...
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getMe "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/setWebhook "HTTP/1.1 200 OK"
INFO:telegram.ext.Application:Application started
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:env_config:Created new user record for 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Document received: viralboost-a9b2c-firebase-adminsdk-osgd2-af2a69fa3a.json (application/json)
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/answerCallbackQuery "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/getFile "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: GET https://api.telegram.org/file/bot7580016374%3AAAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/documents/file_7.json "HTTP/1.1 200 OK"
INFO:env_config:Calling API: None
ERROR:env_config:API error: Invalid type for url.  Expected str or httpx.URL, got <class 'NoneType'>: None
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/editMessageText "HTTP/1.1 200 OK"
INFO:env_config:Media received from user 6133968140
INFO:env_config:Upserting user 6133968140
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
INFO:env_config:Document received: ADIS-Modules-1.pdf (application/pdf)
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/deleteMessage "HTTP/1.1 200 OK"
INFO:httpx:HTTP Request: POST https://api.telegram.org/bot7580016374:AAGdyvJ5NZqpUIC3-pJipohrgJeTH4C6owA/sendMessage "HTTP/1.1 200 OK"
